docs/percona-monitoring-and-management-doc-template.md

```diff
@@ -0,0 +1,66 @@
+
+# Feature Name
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+## What is feature_name?
+
+!!! note alert alert-primary "Note"
+    Replace **feature_name** with the name of the feature.
+
+
+Describe the feature.
+
+## Prerequisites
+List the requirements before installing/configuring/using the feature.
+
+**Example**
+
+Install a particular software or utility.
+
+Configure the software or utility.
+
+## Before you begin
+
+Do specific settings in PMM for the feature to work as expected.
+
+
+## Installing/Configuring/Using feature_name
+
+!!! note alert alert-primary "Note"
+    Replace **feature_name** with the name of the feature.
+
+Use numbered list to provide step by step instructions to install/confiigure/use the feature. Use just one instruction per step for clarity.
+
+ Use images (optional)
+
+![!](../_images/image_name.png)
+
+
+**Note, Info (Optional)**
```

Comment:
Pay attention to Notes/Disclaimers/Important/Caution etc. Instead of placing the information from the Note etc above or below a procedure, integrate the information in the Note into the procedure itself.

What we want is to signpost important information, not to bury it under a Notes label. Users tend to overlook Notes section, especially when there are a few on the page.